slitlessutils/core/preprocess/crrej/drizzle.py

```diff
@@ -0,0 +1,70 @@
+from drizzlepac import astrodrizzle
+from pathlib import Path
+
+# Args taken from INS HSTaXe FullFrame Cookbooks
+COMMON_ARGS = {
+    'ir': {
+        'mdriztab': True,
+        'preserve': False,
+        'skysub': False,
+        'final_fillval': None
+    },
+    'uvis': {
+        'mdriztab': True,
+        'preserve': False,
+        'skysub': True,
+        'skymethod': 'match',
+        'final_fillval': None
+    },
+    'acs': {
+        'build': True,
+        'mdriztab': True,
+        'in_memory': False,
+        'skysub': False,
+        'driz_separate': False,
+        'median': False,
+        'blot': False,
+        'driz_cr': False,
+        'driz_sep_wcs': False,
+        'driz_combine': True,
+        'final_wcs': False
+    }
+}
+
+
+def drizzle(files, instrument=None, outdir=Path().absolute(), **kwargs):
+    '''
+    Runs AstroDrizzle as part of Cosmic Ray handling. Passes any additional arguments
+    not listed here directly to AstroDrizzle. Any user-specified arguments will override
+    
+
+    Parameters
+    ----------
+    files : str to a file catalog, or a python list of files
+        The list of file to be processed by AstroDrizzle
+    
+    instrument : str (one of 'ir', 'uvis', 'acs')
```

Comment:
```suggestion
    instrument : {'ir', 'uvis', 'acs'}

Reply:
Implemented in https://github.com/spacetelescope/slitlessutils/pull/27/commits/ac2852297eea3c1651a549f0d7f8872551635cd0